wikimedia/wikipedia-ios
# Bug fix - Native Editor template selection fix

**Phabricator:** 
https://phabricator.wikimedia.org/T348074

### Notes
This fixes a bug when toggling templates when the template contained formatted text. Toggling off and on worked a couple of times, then would only insert, unlike templates that did not contain formatted text.

### Test Steps
1. On Staging, go to an EN article with a short description. Toggle one of the short description works as bold.
2. Now toggle off templates and confirm the outer {{ }} are removed.
3. Now toggle back on. Outer {{ }} are added.
4. Now toggle back off. Outer {{ }} are removed again.

### Screenshots
Before:
![out1](https://github.com/wikimedia/wikipedia-ios/assets/3620196/993741d7-63ed-4930-8ecc-9475c52cd9d9)

After:
![out3](https://github.com/wikimedia/wikipedia-ios/assets/3620196/59257c50-8cbd-43bd-a60d-e16e42874ee5)
